pkg/kubecost/asset.go

```diff
@@ -2838,6 +2838,31 @@ func (as *AssetSet) accumulate(that *AssetSet) (*AssetSet, error) {
 	return acc, nil
 }
 
+type Diff struct {
+	Asset Asset
```

Comment:
What do you think about making this generic (`Diff[T]` with `entity T`) or general (`Diff` with `name string`)? Anything else we should include here, like change in cost or window before/after?

Be mindful that we may want to run diffs between large data sets, so there will be a tradeoff between having the entire Asset (or T) and having only the relevant fields.

Reply:
I like the idea of making the Diff struct take in generic types. If performance becomes a concern here this can certainly be changed, but I would like to get an idea of how this feature is going to be used.

Regarding a change in cost, I plan to add that functionality in future iterations.